database/mssql/scripts/dbscripts/PSP_PIMS_S10_00/Build/82_DML_LEASE_TESTING.sql

```diff
@@ -0,0 +1,55 @@
+/* -----------------------------------------------------------------------------
+Insert lease data for testing
+. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 
+Author        Date         Comment
+------------  -----------  -----------------------------------------------------
+Doug Filteau  2021-Aug-19  Initial version
+----------------------------------------------------------------------------- */
+
+-- Insert addresses
+INSERT INTO PIMS_ADDRESS (STREET_ADDRESS_1, STREET_ADDRESS_2, STREET_ADDRESS_3, MUNICIPALITY_NAME, POSTAL_CODE, PROVINCE_STATE_ID, COUNTRY_ID, ADDRESS_USAGE_TYPE_CODE, APP_CREATE_TIMESTAMP, APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_LAST_UPDATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_DIRECTORY)
+VALUES
+  (N'45 - 904 Hollywood Crescent', N'Living in a van', N'Down by the River',                N'Hollywood North', N'M4T F0L', 1, 1, N'PHYSADDR', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data'),
+  (N'PO Box 2',                    N'Stealth Camping', N'Walmart Parking Lot',              N'South Podunk',    N'H1I B0B', 1, 1, N'PHYSADDR', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data'),
+  (N'123 Main Street',             N'PO Box 123',      N'Next to the Dairy Queen',          N'East Podunk',     N'I4M B0B', 1, 1, N'PHYSADDR', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data'),
+  (N'456 Souris Street',           N'PO Box 250',      N'A Hoot and a holler from the A&W', N'North Podunk',    N'IH8 B0B', 1, 1, N'PHYSADDR', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data');
+
+-- Insert organiations
+INSERT INTO PIMS_ORGANIZATION (ORGANIZATION_IDENTIFIER, ORGANIZATION_NAME, ORGANIZATION_TYPE_CODE, ORG_IDENTIFIER_TYPE_CODE, ADDRESS_ID, APP_CREATE_TIMESTAMP, APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_LAST_UPDATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_DIRECTORY)
```

Comment:
This is missing IS_DISABLED, which is non-nullable.

Reply:
This causes the insert to fail in SSMS

Reply:
IS_DISABLED defaults to FALSE so I didn't feel the need to add to the SQL.  If you need it, let me know.

Reply:
IS_DISABLED defaults to FALSE so I didn't feel the need to add to the SQL.  If you need it, let me know.

Reply:
IS_DISABLED defaults to FALSE so I didn't feel the need to add to the SQL.  If you need it, let me know.

Reply:
The script fails in the current version of the database, so I will need it.

Reply:
From the looks of things the current version of the database doesn't support these default values.